hibernate-search-elasticsearch/src/main/java/org/acme/hibernate/search/elasticsearch/model/Book.java

```diff
@@ -17,8 +17,8 @@
 @Indexed
 public class Book extends PanacheEntity {
 
-    @FullTextField(analyzer = "stemmingAnalyzer")
-    @KeywordField(name = "title_sort", sortable = Sortable.YES, normalizer = "defaultNormalizer")
+    @FullTextField(analyzer = "english")
+    @KeywordField(name = "title_sort", sortable = Sortable.YES, normalizer = "sort")
```

Comment:
I think you removed "title_sort" from the guide?

Reply:
Ah yes, good catch.